### DISCUSSION TITLE
Allow for Implementing Display Names for Server Services

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#336] [Suggestions]
This has to do with implementing display names for service servers, like Syncthing installed through GSA. That way it'd be a little better for organization. 

![image](https://user-images.githubusercontent.com/44717221/215411451-6ed5fc62-450d-40de-8637-fbc5e62e7ef1.png)

**Comment by mvaessen** [comment]
Thanks for your suggestion! This will be available in the upcoming update
